QWeb/internal/js/get_all_dropdown_elements_shadow_dom.js

```diff
@@ -0,0 +1,14 @@
+// get_all_dropdown_elements_from_shadow_dom.js
+// Finds all select elements in shadow DOM
+// Assumes recursiveWalk is defined in the context
```

Comment:
Could be useful to tell in the comment where to find the `recursiveWalk`, add to other `recursiveWalk` depending sources if you think it's useful
`// Assumes recursiveWalk (get_recursive_walk.js) is defined in the context`

Reply:
ok